.github/CODEOWNERS

```diff
@@ -3,62 +3,62 @@
 # Please define less specific codeowner paths before more specific codeowner paths in order for the more specific rule to have priority
 
 # Root
-* @smartcontractkit/foundations
+* @smartcontractkit/core
 
-.changeset @smartcontractkit/releng @smartcontractkit/foundations
+.changeset @smartcontractkit/releng @smartcontractkit/core
 
 # Chains
-/common @smartcontractkit/bix-framework
-/core/chains/ @smartcontractkit/bix-framework
```

Comment:
Why are we changing ownership of `/core/chains` from `bix-framework` to `core`? This code is owned by bix-framework

Reply:
(At least, `/core/chains/evm` not entirely sure about the rest)

Reply:
This is extending the possible approvers, core being co-owners

Reply:
Oh, lol! Sorry, yeah that makes sense